aptos-move/aptos-vm/src/sharded_block_executor/cross_shard_state_view.rs

```diff
@@ -179,3 +180,55 @@ mod tests {
         wait_thread.join().unwrap();
     }
 }
+
+pub const TOTAL_SUPPLY_AGGR_BASE_VAL: u128 = u128::MAX >> 1;
+#[derive(Clone)]
+pub struct CrossShardStateViewAggrOverride<'a, S> {
```

Comment:
It will be better if this implementation is generic i.e, it just stores a StateView base class instead of `CrossShardStateView`, and rename this to something like `AggregatorOverriddenStateView`

Reply:
done